ui/src/pages/publications/[id]/index.tsx

```diff
@@ -502,17 +521,17 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
                             <h1 className="col-span-7 mb-4 block font-montserrat text-2xl font-bold leading-tight text-grey-800 transition-colors duration-500 dark:text-white-50 md:text-3xl xl:text-3xl xl:leading-normal">
                                 {publicationData.title}
                             </h1>
-                            {isBookmarkVisible && (
+                            {isBookmarkButtonVisible && (
                                 <div className="col-span-1 grid justify-items-end">
                                     <button
                                         className="h-8 hover:cursor-pointer focus:outline-none focus:ring focus:ring-yellow-200 focus:ring-offset-2 dark:outline-none dark:focus:ring dark:focus:ring-yellow-600 dark:focus:ring-offset-1"
                                         onClick={onBookmarkHandler}
                                         aria-label="toggle-bookmark"
-                                        title={`${isBookmarked ? 'Remove bookmark' : 'Bookmark this publication'}`}
+                                        title={`${showBookmark ? 'Remove bookmark' : 'Bookmark this publication'}`}
```

Comment:
```suggestion
                                        title={`${isBookmarked ? 'Remove bookmark' : 'Bookmark this publication'}`}
```

we can replace with 'isBookmarked'